.gitpod.yml

```diff
@@ -3,9 +3,14 @@ additionalRepositories:
   - url: https://github.com/mattermost/mattermost-webapp
 
 ports:
+  # 8065 - for Mattermost instance
   - port: 8065
     onOpen: open-browser
     visibility: public
+  # 6080 - for noVNC instance (Cypress)
+  - port: 6080
+    onOpen: open-browser
+    visibility: public
```

Comment:
I'm not sure if we want to have the `onOpen` here. There are lots of use cases why someone would be opening a Mattermost repository in Gitpod, and writing/editing e2e tests are not always relevant to their use case, so this could cause a bit of noise or confusion having this automatically open.

Reply:
Would you be amenable to maybe changing `onOpen: open-browser` to either `onOpen: ignore` or `onOpen: notify` (the default)?

Reply:
Thinking more about this, what does setting this in `.gitpod.yml` change, as far as how the port is used? The port should still be accessible without setting this here. Is this added for convenience instead, or are you not able to access the virtual desktop otherwise?

Reply:
I realize now that we should put in this file all ports that get used by the workspace. This allows us to do `onOpen: ignore` for port `5432` for instance. This way there's less noise when arriving at the workspace.

I think we should do this change you propose, and use `onOpen: notify` for port `6080`. What do you think @simcard0000?

Reply:
@nickmister I added it for convenience; to be honest I didn't even know `6080` was the port for the virtual desktop until I messed around a bit. `visibility: public` as far as I know isn't necessary; but also all ports anyways are public. 

I do agree that putting all ports used by the workspace in here and adding `onOpen: ignore` for most of them would de-clutter the workspace (and would be a good place to comment on what each port is for). I'll change `onOpen` for `6080` and add the rest of the ports.